js/interface.js

```diff
@@ -1811,7 +1811,15 @@ function configureAddRuleUI(rule) {
     showAutocompleteOnFocus: true
   });
 
-  $('input[name="exclude"]').tokenfield('setTokens', rule.exclude ? rule.exclude : rule.include ? rule.include : []);
+  var tokenField = [];
+
+  if (rule.exclude) {
+    tokenField = rule.exclude;
+  } else {
+    tokenField = rule.include;
```

Comment:
doesn't this end up with `undefined` instead of an empty array when none of the rule include/exclude are defined?

Reply:
changes done as per suggestion
cc: @NikitaJadhavXoriant